bindings/nodejs/CHANGELOG.md

```diff
@@ -19,6 +19,13 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Security -->
 
+## 1.0.2 - 2023-MM-DD
+
+### Fixed
+
+- Constrcutor types in `RegularTransactionEssence`;
```

Comment:
```suggestion
- Constructor types in `RegularTransactionEssence`;
```

Reply:
I had the feeling I misspelled this word. But I ran my quick translator on it, and it translated it correctly. I'll never use the translator as spell checker anymore 😆